deploy/backplane/20-cssre.SubjectPermission.yml

```diff
@@ -10,7 +10,6 @@ spec:
   permissions:
   - allowFirst: true
     clusterRoleName: backplane-cssre-admins-project
-    namespacesAllowedRegex: .*
-    namespacesDeniedRegex: "(^kube$|^kube-.*|^openshift$|^openshift-.*|^default$)"
+    namespacesAllowedRegex: "(^redhat-rhmi-.*|^redhat-rhoam-.*)"
```

Comment:
@alanmoran We had a meeting with sre-p to discuss backplane permissions, and these changes were merged for backplane.  I think we'll need to revisit these since you implied we need to move back to the previous 'deny' model?

Reply:
So the issue is the walkthroughs and fuse afaik. When an RHMI user interacts with them they generate random namespaces based on the user feedback.  I'm not sure we actually need access to these?(do you know?)

eg
![image](https://user-images.githubusercontent.com/1325145/100314817-67ab6480-2faf-11eb-89ef-f0a3999b11bb.png)
and
![image](https://user-images.githubusercontent.com/1325145/100314877-83166f80-2faf-11eb-8c47-8338d1f7bbf5.png)